sequelize/data/entity.json

```diff
@@ -9,6 +9,10 @@
         "state": "MD",
         "zip": "12345"
       },
+      "latlng": [
```

Comment:
Let's add this as part of the address column. That way we don't need an extra column for it.